src/muya/lib/contentState/backspaceCtrl.js

```diff
@@ -115,15 +117,17 @@ const backspaceCtrl = ContentState => {
 
   ContentState.prototype.backspaceHandler = function (event) {
     const { start, end } = selection.getCursorRange()
+    const { selectedImage } = this
 
     if (!start || !end) {
       return
     }
 
     // handle delete selected image
-    if (this.selectedImage) {
+    if (selectedImage) {
       event.preventDefault()
-      return this.deleteImage(this.selectedImage)
+      this.selectedImage = null
```

Comment:
It's no need to to set `this.selectedImage` to `null` if we already set `this.selectedImage = null` in `deleteImage` method?

Reply:
If you prefer a commit with only ImageCtrl.js fixes, re-create a another pull request.
Should I close this pull request once?